tests/zfs-tests/cmd/mkfile/mkfile.c

```diff
@@ -44,7 +44,13 @@
 
 #define	FILE_MODE	(S_ISVTX + S_IRUSR + S_IWUSR)
 
-static _Noreturn void usage(void);
+static __attribute__((noreturn)) void
+usage()
```

Comment:
Changing `usage()` -> `usage(void)` will get rid of the current error in the builds:
`mkfile.c:48:1: error: function declaration isn’t a prototype [-Werror=strict-prototypes]`

Reply:
Fixed